### PR TITLE
Update branding to 9.0-alpha1

### PR DESCRIPTION
**Pull request by wtgodbe**
To be merged after we snap for rc1

### CODE REVIEW

**Comment by BrennanConroy** [comment]
Does the site extension need to be updated to include 8.0 or is that something we do later?

**Comment by wtgodbe** [comment]
> Does the site extension need to be updated to include 8.0 or is that something we do later?

Typically we do that after we ship the RTM version, along with a bunch of other cleanup

File: eng/Versions.props [hunk 1]
```diff
@@ -17,8 +17,8 @@
     -->
     <StabilizePackageVersion Condition="'$(StabilizePackageVersion)' == ''">false</StabilizePackageVersion>
     <DotNetFinalVersionKind Condition="'$(StabilizePackageVersion)' == 'true'">release</DotNetFinalVersionKind>
-    <PreReleaseVersionLabel>rc</PreReleaseVersionLabel>
-    <PreReleaseBrandingLabel>Release Candidate $(PreReleaseVersionIteration)</PreReleaseBrandingLabel>
+    <PreReleaseVersionLabel>alpha</PreReleaseVersionLabel>
```
Comment (BrennanConroy) [inline eng/Versions.props:20]: Any reason we use alpha when our first release is preview1?

Comment (wtgodbe) [reply eng/Versions.props:20]: So we don't accidentally message that we're getting ready to ship a preview1. We usually don't brand for preview1 until around February

**Review by BrennanConroy — APPROVED** [review]

**Pull request merged**